recipes/msaboot/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: msaboot
+  version: "0.1.0"
+
+source:
+  url: https://github.com/phac-nml/msaboot/archive/v0.1.0.tar.gz
+  sha256: 41a34899b4c768abec4bf4e15d5a0a08451208fab9403b00454f1b7203971fa6
+
+build:
+  number: 0
+  skip: True # [win]
+
+requirements:
+  build:
+    - python
+    - numpy
+    - biopython
+
+  run:
+    - python
+    - numpy
+    - biopython
+
+test:
+  commands:
+    - msaboot -h | grep "bootstraps FASTA input data"
+
+about:
+  home: https://github.com/phac-nml/msaboot
+  license: Apache 2.0
```

Comment:
Can you also add
```yaml
  license_file: LICENSE.txt
```

Reply:
Thank you, Please let me know if there is anything else I should change.